docs/pbc/all/payment-service-provider/202311.0/base-shop/third-party-integrations/stripe/install-and-configure-stripe-prerequisites.md

```diff
@@ -193,6 +193,30 @@ use Spryker\Zed\SalesPayment\Communication\Plugin\Oms\SendCancelPaymentMessageCo
 
 ```
 
+6. In `src/Pyz/Zed/Payment/PaymentDependencyProvider.php`, add or update the following plugins:
+
+
+```php
+// ...
+
+use Spryker\Zed\OauthClient\Communication\Plugin\Payment\AccessTokenPaymentAuthorizeRequestExpanderPlugin;
+
+
+    // ...
+
+    /**
+     * @return array<int, \Spryker\Zed\PaymentExtension\Dependency\Plugin\PaymentAuthorizeRequestExpanderPluginInterface>
+     */
+    protected function getPaymentAuthorizeRequestExpanderPlugins(): array
+    {
+        return [
+            //...
+            new AccessTokenPaymentAuthorizeRequestExpanderPlugin(),
+        ];
+    }
+
+```
+
```

Comment:
I believe it's only a quick fix and won't solve other potential problems.

So please check what was changed since prev. SCOS 202009 till now. Compare two https://github.com/spryker-shop/b2c-demo-shop/compare/202009.0...202311.0 i

Reply:
This is the only change in the PaymentDependencyProvider. I can't review more than 1.5k files and try to figure out what has changed. Do you have anything specific in mind we should check?